docs/03.API-EXAMPLE.md

```diff
@@ -180,36 +163,21 @@ main (int argc, char * argv[]) {
 
 The sample will also output 'Hello, World!'. However, now it is not just a part of the source script, but the value, dynamically supplied to the engine.
 
-## Step 5. Description of JavaScript value descriptors
-
-Structure, used to put values to or receive values from the engine is the following:
-
-- `type` of the value:
-  - JERRY_API_DATA_TYPE_VOID (void);
-  - JERRY_API_DATA_TYPE_UNDEFINED (undefined);
-  - JERRY_API_DATA_TYPE_NULL (null);
-  - JERRY_API_DATA_TYPE_BOOLEAN (true / false);
-  - JERRY_API_DATA_TYPE_FLOAT32 (number);
-  - JERRY_API_DATA_TYPE_FLOAT64 (number);
-  - JERRY_API_DATA_TYPE_UINT32 (number);
-  - JERRY_API_DATA_TYPE_STRING (string);
-  - JERRY_API_DATA_TYPE_OBJECT (object reference);
-- `v_bool` (if JERRY_API_DATA_TYPE_BOOLEAN) - boolean value;
-- `v_float32` (if JERRY_API_DATA_TYPE_FLOAT32) - float value;
-- `v_float64` (if JERRY_API_DATA_TYPE_FLOAT64) - double value;
-- `v_uint32` (if JERRY_API_DATA_TYPE_UINT32) - 32 bit unsigned integer value;
-- `v_string` (if JERRY_API_DATA_TYPE_STRING) - pointer to string;
-- `v_object` (if JERRY_API_DATA_TYPE_OBJECT) - pointer to object.
-
-Abstract values, to be sent to or received from the engine are described with the structure.
+## Step 5. Description of JerryScript value descriptors
 
-Pointers to strings or objects and values should be released just when become unnecessary, using `jerry_release_string` or `jerry_release_object` and `jerry_release_value`, correspondingly.
+JerryScript value can be a boolean, number, null, object, string or undefined. The value has an error flag,
+that indicates whether is an error or not. Every type has an error flag not only objects.
```

Comment:
It should also be noted that the error flag should be cleared before the value is passed as an argument.